File: src/3038.maximum-number-of-operations-with-the-same-score-i.3320/solution.ts
function maxOperations(nums: number[]): number {
    const S = nums[0] + nums[1];
    let result = 1;
    for (let i = 2; i < nums.length; i += 2) {
        if (i + 1 === nums.length) {
            break;
        }
        if (nums[i] + nums[i + 1] === S) {
            result++
        } else {
            break;
        }
    }
    return result
};

File: src/3038.maximum-number-of-operations-with-the-same-score-i.3320/solution.java
class Solution {
    public int maxOperations(int[] nums) {
        final int score = nums[0]+nums[1];
        int cnt = 1;
        for(int i=2;i+1<nums.length;i+=2){
            if(nums[i]+nums[i+1] == score){
                cnt++;
            }else{
                break;
            }
        }
        return cnt;
    }
}